media/plg_system_tour/js/guide.js

```diff
@@ -0,0 +1,70 @@
+Joomla = window.Joomla || {};
+(function (Joomla, window) {
+    document.addEventListener('DOMContentLoaded', function() {
+        var urlParameters = Joomla.getOptions('tour-guide');
+        if (!urlParameters) { return; }
+            if (window.location.href.indexOf("&") > -1) {
+                var filename = '';
+                var URL = '';
+
+                if (urlParameters.urlOption)  //case if their is presence of two paramteres and one of them is option
+                {
+                    filename = urlParameters.urlOption;
+                    if (urlParameters.urlView)         //case if their is presence of two paramteres and they are view and option
+                    {
+                        filename = filename + '_' + urlParameters.urlView;
+                    }
+                    if ((window.location.href.indexOf("layout=") > 0))   //case if their is presence of three paramteres and they are view and option and layout
+                    {
+                        filename = filename + '_' + urlParameters.urlLayout;
+                    }
+                    filename = filename + '.' + 'json';//made the file name to be fetched out
+                    URL = urlParameters.urlOption + '/' + filename; // path from where the file to be fetched
+                }
+            }
+            else {
+                filename = urlParameters.urlOption + '.' + 'json';
+                URL = urlParameters.urlOption + '/' + filename;
+            }
+        var btn= document.createElement('button');
+        btn.classList.add('btn');
+        btn.classList.add('btn-sm');
+        btn.classList.add('btn-outline-primary');
+        btn.setAttribute('id', 'startTourBtn');
+        btn.innerHTML = '<span class="icon"></span>Tour_Vanilla</button>';
+        document.getElementById('toolbar').appendChild(btn);
+        Joomla.request(  {
+            url: window.location.protocol + '//' + window.location.host + '/' + window.location.pathname.split('/')[1] + '/media/guide/' + urlParameters.langtag + '/'+ URL,
+            method: 'GET',
+            data:    '',
+            perform: true,
+            onSuccess: function(data) {
```

Comment:
@Hutchy68 If you have the concern regarding the Button name it is not yet fixed.It will be decided.
And regarding the code, Dimitris Suggested for Pure JS